starter-code/src/app/movie-service/movie.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import moviesList, {Movie} from '../movie-service/movies-list';
+
+@Injectable()
+export class MovieService {
+  moviesList: Array<Movie> = [...moviesList];
```

Comment:
I like your use of ES6, but I think in this case you are creating unnecessary computation. Since movieList is already the array you need, you are essentially just creating a copy of it, which behind the scenes is looping over the array for no reason. On a larger scale, if there were 10K movies, it would be an extra 10K actions by the engine :)
In this case I would just stick with `moviesList: Array<Movie> = moviesList`